src/extensions/surveys/components/QuestionHeader.tsx

```diff
@@ -23,7 +23,7 @@ export function QuestionHeader({
             <div className="survey-question">{question}</div>
             {description &&
                 renderChildrenAsTextOrHtml({
-                    component: h('div', { className: 'description' }),
+                    component: h('div', { className: 'survey-question-description' }),
```

Comment:
makes our survey question descriptions easier to query in the DOM, see: https://github.com/PostHog/posthog/pull/24143